Japanese localization of the DeployFromSettings page

Yamagami-Takuma opened the pull request:
Japanese localization of the DeployFromSettings page.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubernetes/dashboard/818)
<!-- Reviewable:end -->

Yamagami-Takuma commented:
@f-higashi Please review this! :)

f-higashi commented:
Review status: 0 of 1 files reviewed at latest revision, 10 unresolved discussions.

---

*[i18n/messages-ja.xtb, line 31 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZN96jJEnNfVj_Ox9v:-KJZN96jJEnNfVj_Ox9w:1122029080) ([raw file](https://github.com/kubernetes/dashboard/blob/f99d82894853fa50131b238033129c5cdbf0c5f1/i18n/messages-ja.xtb#L31)):*
> ```
> 	<translation id="3379471811475022108" key="MSG_DEPLOY_SETTINGS_APP_NAME_PATTERN_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears when the app name input on the deploy from settings page does not match the expected pattern.">小文字の半角英字で始めてください。その後は小文字の半角英字・数字・文中のハイフンが使用できます。</translation>
> 	<translation id="7213646373445455290" key="MSG_DEPLOY_SETTINGS_APP_NAME_MAX_LENGTH_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears when the typed in app name on the deploy from settings page exceeds the maximal allowed length.">最大 <ph name="MAX_LENGTH" /> 文字です</translation>
> 	<translation id="9171860855281312734" key="MSG_DEPLOY_SETTINGS_APP_NAME_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the `App name` input on the deploy from settings page.">この名前のアプリケーションラベルをレプリケーションコントローラーとサービスに追加してデプロイします。</translation>
> ```

In this sentence, 'app' is label key.  It should not be translated.
How about "このバリューをもつ'app'ラベルが、配備されるレプリケーションコントローラ
とサービスに付与されます。

---

*[i18n/messages-ja.xtb, line 35 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZOpAhJ2gF7sL2dkBc:-KJZOpAi_kaxgiBWYpxy:82783891) ([raw file](https://github.com/kubernetes/dashboard/blob/f99d82894853fa50131b238033129c5cdbf0c5f1/i18n/messages-ja.xtb#L35)):*
> ```
> 	<translation id="2384273649400241565" key="MSG_DEPLOY_SETTINGS_CONTAINER_IMAGE_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Container image&quot;, which appears as a placeholder in an empty input field for a container image on the deploy from settings page.">コンテナイメージ</translation>
> 	<translation id="5045196368183048329" key="MSG_DEPLOY_SETTINGS_CONTAINER_IMAGE_REQUIRED_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the container image input on the deploy from settings page is required.">必須項目です</translation>
> 	<translation id="3987564063412539284" key="MSG_DEPLOY_SETTINGS_CONTAINER_IMAGE_INVALID_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the typed in container image is invalid. This text must end with a colon.">コンテナイメージが無効です:</translation>
> ```

'不正' is often-used for invalid
How about  'コンテナイメージが不正な値です:'

---

*[i18n/messages-ja.xtb, line 42 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZTvl9gWcRC6aamhNl:-KJZTvlAktwmPViZe9Zc:-435022536) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L42)):*
> ```
> 	<translation id="6564510320385038864" key="MSG_DEPLOY_SETTINGS_NUMBER_OF_PODS_HIGH_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears as a warning when the user has specified a really high number of pods on the deploy from settings page.">多数のポッドを設定すると、クラスターとダッシュボードにパフォーマンスの問題が発生する可能性があります。</translation>
> 	<translation id="3826796659710276505" key="MSG_DEPLOY_SETTINGS_NUMBER_OF_PODS_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the &quot;number of pods&quot; input on the deploy from settings page.">設定したポッド数を保つために、レプリケーションコントローラーが作成されます。</translation>
> 	<translation id="5840124104405813335" key="MSG_DEPLOY_SETTINGS_PORT_MAPPINGS_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the &quot;port mappings&quot; input on the deploy from settings page.">ポートはオプションです。指定した場合、コンテナから見えるターゲットポートに、ポート（受信）がマッピングされたサービスを作成します。</translation>
> ```

It is better '指定した場合、コンテナから見えるターゲットポートに、ポート（受信）がマッピングされたサービスが作成されます。’

---

*[i18n/messages-ja.xtb, line 49 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZUmnjQVw6NoQCk3sX:-KJZUmnjQVw6NoQCk3sY:-583943031) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L49)):*
> ```
> 	<translation id="302338638356502088" key="MSG_DEPLOY_SETTINGS_LABELS_KEY_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Key&quot; for the key input fields, in the labels section on the deploy from settings page.">キー</translation>
> 	<translation id="4605907629737798248" key="MSG_DEPLOY_SETTINGS_LABELS_VALUE_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Value&quot; for the value input fields, in the labels section on the deploy from settings page.">バリュー</translation>
> 	<translation id="7004903789599705230" key="MSG_DEPLOY_SETTINGS_LABELS_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the &quot;Labels&quot; section on the deploy from settings page.">指定したラベルを、作成したレプリケーションコントローラー、サービス（もしあれば）、ポッドに適用します。共通のラベルは、リリース、環境、階層、パーティション、トラックを含みます。</translation>
> ```

release, environment, tier, partition and track are example of labels.
How  about "一般的なラベルは、'release', 'environment', .... です。"

---

*[i18n/messages-ja.xtb, line 52 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZakKr_fFkXHelkyXE:-KJZakKr_fFkXHelkyXF:566918819) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L52)):*
> ```
> 	<translation id="9130827171778144236" key="MSG_DEPLOY_SETTINGS_NAMESPACE_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Namespace&quot; label, for the namespace selection box on the deploy from settings page.">ネームスペース</translation>
> 	<translation id="1799676332581487715" key="MSG_DEPLOY_SETTINGS_NAMESPACE_CREATE_ACTION" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="The text appears in the namespace selection box on the deploy from settings page, and acts as a button. Clicking it creates a new namespace.">新規ネームスペースの作成...</translation>
> 	<translation id="5947931806556909716" key="MSG_DEPLOY_SETTINGS_NAMESPACE_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the namespace selection on the deploy from settings page.">ネームスペースによって、リソースを論理的なグループに分割します。</translation>
> ```

named is missed.
How about '論理的に名前付きグループに分割されます'

---

*[i18n/messages-ja.xtb, line 55 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZYUajHE1YgFW41ZWF:-KJZYUakxgxhXcEZdxTN:-91882391) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L55)):*
> ```
> 	<translation id="7821863265656538668" key="MSG_DEPLOY_SETTINGS_IMAGE_PULL_SECRET_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Image Pull Secret&quot;, which appears as a placeholder in the image pull secret selection box on the deploy from settings page.">イメージプルシークレット</translation>
> 	<translation id="6045084755231450864" key="MSG_DEPLOY_SETTINGS_IMAGE_PULL_SECRET_CREATE_ACTION" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="The text appears in the image pull secret selection box on the deploy from settings page and acts as a button. Pressing it creates a new image pull secret.">新規シークレットの作成...</translation>
> 	<translation id="5111944609518832752" key="MSG_DEPLOY_SETTINGS_IMAGE_PULL_SECRET_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the &quot;Image Pull Secret&quot; selection box on the deploy from settings page.">指定したイメージがプライベートの場合、プルシークレットのクレデンシャルを要求されることがあります。既存のシークレットを選択するか、新規シークレットを作成してください。</translation>
> ```

I like '認証情報'  than クレデンシャル

---

*[i18n/messages-ja.xtb, line 56 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZcrrLP_Vl7TqaVhr0:-KJZcrrLP_Vl7TqaVhr1:-87566998) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L56)):*
> ```
> 	<translation id="6045084755231450864" key="MSG_DEPLOY_SETTINGS_IMAGE_PULL_SECRET_CREATE_ACTION" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="The text appears in the image pull secret selection box on the deploy from settings page and acts as a button. Pressing it creates a new image pull secret.">新規シークレットの作成...</translation>
> 	<translation id="5111944609518832752" key="MSG_DEPLOY_SETTINGS_IMAGE_PULL_SECRET_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the &quot;Image Pull Secret&quot; selection box on the deploy from settings page.">指定したイメージがプライベートの場合、プルシークレットのクレデンシャルを要求されることがあります。既存のシークレットを選択するか、新規シークレットを作成してください。</translation>
> 	<translation id="4718361914515511196" key="MSG_DEPLOY_SETTINGS_CPU_REQUIREMENT_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;CPU requirement&quot;, which appears as a placeholder for the cpu input on the deploy from settings page.">CPU (コア数)</translation>
> ```

'requirement' is missed.
CPU要件

---

*[i18n/messages-ja.xtb, line 59 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZcyjBNwW5ggUan1o9:-KJZcyjBNwW5ggUan1oA:363894619) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L59)):*
> ```
> 	<translation id="6710871378232724793" key="MSG_DEPLOY_SETTINGS_CPU_NOT_A_NUMBER_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the typed in CPU cores count on the deploy page is not a number.">数字を入力してください。</translation>
> 	<translation id="5490209057800683483" key="MSG_DEPLOY_SETTINGS_CPU_NEGATIVE_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the typed in number of CPU cores (on the deploy page) cannot be negative.">正数を入力してください。</translation>
> 	<translation id="6150169179901412462" key="MSG_DEPLOY_SETTINGS_MEMORY_REQUIREMENT_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Memory requirement&quot;, which appears as a placeholder for the memory input on the deploy from settings page.">メモリー (MiB)</translation>
> ```

'requirement' is missed.
メモリー要件

---

*[i18n/messages-ja.xtb, line 62 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZZZKNvE1ndNiI5qji:-KJZZZKNvE1ndNiI5qjj:1839703955) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L62)):*
> ```
> 	<translation id="6950575202934220010" key="MSG_DEPLOY_SETTINGS_MEMORY_NOT_A_NUMBER_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the typed in memory on the deploy page is not a number.">数字を入力してください。</translation>
> 	<translation id="7214715359451365159" key="MSG_DEPLOY_SETTINGS_MEMORY_NEGATIVE_WARNING" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Appears to tell the user that the typed in memory (on the deploy page) cannot be negative.">正数を入力してください。</translation>
> 	<translation id="8647847835448929594" key="MSG_DEPLOY_SETTINGS_CPU_MEM_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the memory and cpu requirement inputs on the deploy from settings page.">最小の CPU とメモリーをコンテナに指定できます。</translation>
> ```

'requirement' is missed.

---

*[i18n/messages-ja.xtb, line 64 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZdVKiUL7ath3txDy4:-KJZdVKiUL7ath3txDy5:363330293) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L64)):*
> ```
> 	<translation id="8647847835448929594" key="MSG_DEPLOY_SETTINGS_CPU_MEM_USER_HELP" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="User help for the memory and cpu requirement inputs on the deploy from settings page.">最小の CPU とメモリーをコンテナに指定できます。</translation>
> 	<translation id="2530547307291673869" key="MSG_DEPLOY_SETTINGS_RUN_COMMAND_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Run command&quot; (a noun, not a verb), which serves as a placeholder for the 'Command' input on the deploy from settings page.">コマンドの実行</translation>
> 	<translation id="8947217119676480958" key="MSG_DEPLOY_SETTINGS_RUN_COMMAND_ARGUMENTS_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Run command arguments&quot;, which serves as a placeholder for the command arguments input on the deploy from settings page.">コマンドの実行の引数</translation>
> ```

It is strange japanese.
How about '実行コマンドの引数'.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/818)*
<!-- Sent from Reviewable.io -->

bryk commented:
@maciaszczykm Please merge when the review is done :)

f-higashi commented:
Review status: 0 of 1 files reviewed at latest revision, 11 unresolved discussions.

---

*[i18n/messages-ja.xtb, line 48 \[r3\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJ_0UzPBm8ujPjt4AEG:-KJ_0UzPBm8ujPjt4AEH:508318621) ([raw file](https://github.com/kubernetes/dashboard/blob/9579d91e87bb155bb571f6191de8c4b70eda36aa/i18n/messages-ja.xtb#L48)):*
> ```
> 	<translation id="215841056414165502" key="MSG_DEPLOY_SETTINGS_LABELS_TITLE" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Title of the labels input section on the deploy from settings page.">ラベル</translation>
> 	<translation id="302338638356502088" key="MSG_DEPLOY_SETTINGS_LABELS_KEY_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Key&quot; for the key input fields, in the labels section on the deploy from settings page.">キー</translation>
> 	<translation id="4605907629737798248" key="MSG_DEPLOY_SETTINGS_LABELS_VALUE_LABEL" source="/home/mirchev/go_workspace/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label &quot;Value&quot; for the value input fields, in the labels section on the deploy from settings page.">バリュー</translation>
> ```

I like '値'

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/818)*
<!-- Sent from Reviewable.io -->

maciaszczykm commented:
@f-higashi Please let me know, when it will be ready for merge.

Yamagami-Takuma commented:
Review status: 0 of 1 files reviewed at latest revision, 11 unresolved discussions.

---

*[i18n/messages-ja.xtb, line 31 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZN96jJEnNfVj_Ox9v:-KJ_2HD69M5Deqk9FjOJ:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/f99d82894853fa50131b238033129c5cdbf0c5f1/i18n/messages-ja.xtb#L31)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> In this sentence, 'app' is label key.  It should not be translated.
> How about "このバリューをもつ'app'ラベルが、配備されるレプリケーションコントローラ
> とサービスに付与されます。
> 
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 35 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZOpAhJ2gF7sL2dkBc:-KJ_2JT7RSuYC6toRwLZ:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/f99d82894853fa50131b238033129c5cdbf0c5f1/i18n/messages-ja.xtb#L35)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> '不正' is often-used for invalid
> How about  'コンテナイメージが不正な値です:'
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 42 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZTvl9gWcRC6aamhNl:-KJ_2KA70kjd4b6LlCgX:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L42)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> It is better '指定した場合、コンテナから見えるターゲットポートに、ポート（受信）がマッピングされたサービスが作成されます。’
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 49 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZUmnjQVw6NoQCk3sX:-KJ_2LidXbwAulq2T5Wg:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L49)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> release, environment, tier, partition and track are example of labels.
> How  about "一般的なラベルは、'release', 'environment', .... です。"
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 52 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZakKr_fFkXHelkyXE:-KJ_2MBEXB8F08fc3V25:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L52)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> named is missed.
> How about '論理的に名前付きグループに分割されます'
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 55 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZYUajHE1YgFW41ZWF:-KJ_2RwohMHpV9ITrwoh:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L55)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> I like '認証情報'  than クレデンシャル
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 56 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZcrrLP_Vl7TqaVhr0:-KJ_2RXU_9CNq4-ZFgrz:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L56)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> 'requirement' is missed.
> CPU要件
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 59 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZcyjBNwW5ggUan1o9:-KJ_2R-5R8ejEutTy2zZ:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L59)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> 'requirement' is missed.
> メモリー要件
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 62 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZZZKNvE1ndNiI5qji:-KJ_2Py6pUa9lwNvVYHJ:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L62)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> 'requirement' is missed.
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 64 \[r2\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJZdVKiUL7ath3txDy4:-KJ_2NphuEzyQgV_KHtX:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/682b28aa036ababebd435b8ed2079b53fec62734/i18n/messages-ja.xtb#L64)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> It is strange japanese.
> How about '実行コマンドの引数'.
</details>
I modified.

---

*[i18n/messages-ja.xtb, line 48 \[r3\]](https://reviewable.io:443/reviews/kubernetes/dashboard/818#-KJ_0UzPBm8ujPjt4AEG:-KJ_2LHGISvOMw0iWeIo:1176402249) ([raw file](https://github.com/kubernetes/dashboard/blob/9579d91e87bb155bb571f6191de8c4b70eda36aa/i18n/messages-ja.xtb#L48)):*
<details><summary><i>Previously, f-higashi (higashi isao) wrote…</i></summary>
> I like '値'
</details>
I modified.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/818)*
<!-- Sent from Reviewable.io -->

f-higashi commented:
LGTM

@maciaszczykm  you can merge this :)
